src/highlight.rs

```diff
@@ -44,6 +45,10 @@ pub static DATA: LazyLock<Data> = LazyLock::new(|| {
     }
 });
 
+static LINK_REGEX: LazyLock<Regex> = LazyLock::new(|| {
+    Regex::new("https?://[^ \"]+").unwrap()
```

Comment:
Perhaps instead of excluding, we can include [valid characters](https://stackoverflow.com/a/1547940)?

Reply:
It is not that simple. For example `'` may not occur at the end of URLs, I think.

Reply:
Hmm …  the current expression still matches the `"`

![Bildschirmfoto vom 2025-01-06 12-45-47](https://github.com/user-attachments/assets/57a4713d-49b7-4043-980f-5cad7aabf777)

Reply:
Right, it was run after html escaping, so it matched on the &quot.

Fixed now with a two-phase replacement.